Description: Install libcomps.pc.
Author: Mihai Moldovan <ionic@ionic.de>

This patch should probably be upstreamed, if necessary in tandem with the
libcomps.pc generation patch.

--- a/libcomps/src/CMakeLists.txt
+++ b/libcomps/src/CMakeLists.txt
@@ -64,5 +64,8 @@ configure_file("${PROJECT_SOURCE_DIR}/..
 install (FILES ${libcomps_HEADERS} DESTINATION include/libcomps)
 install (TARGETS libcomps LIBRARY DESTINATION ${LIB_INST_DIR})
 
+install (FILES "${CMAKE_CURRENT_BINARY_DIR}/libcomps.pc"
+         DESTINATION "${LIB_INST_DIR}/pkgconfig/")
+
 add_subdirectory(python)
 
